code/__defines/misc.dm

```diff
@@ -208,3 +207,8 @@
 #define EMAIL_BROADCAST "broadcast@internal-services.net"
 
 #define LEGACY_RECORD_STRUCTURE(X, Y) GLOBAL_LIST_EMPTY(##X);/datum/computer_file/data/##Y/var/list/fields[0];/datum/computer_file/data/##Y/New(){..();GLOB.##X.Add(src);}/datum/computer_file/data/##Y/Destroy(){. = ..();GLOB.##X.Remove(src);}
+
+#define DEFAULT_SPAWNPOINT_ID "Cryogenic Storage"
```

Comment:
Spawnpoint select work without that hardcode previously

Reply:
You have only 1 lategame spawn point and its actually bad, it can be oxyless or worse. You didnt need that since you have only one, but for future this will be helpful